### PR TITLE
Add dcos-dns metrics registry to telegraf

### PR DESCRIPTION
**Pull request by Fabs**
## High-level description

Add dcos-dns metrics for forwarders.

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS-48336](https://jira.mesosphere.com/browse/DCOS-48336)  Add dcos-dns metrics for forwarders

## Checklist for all PRs

  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here: no tests
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [x] Test Results: [link to CI job test results for component]
  - [x] Code Coverage (if available): [link to code coverage report]

> [dcos/dcos-net@999f63e...3e6119a](https://github.com/dcos/dcos-net/compare/3e6119ae51ac1d457d98ef3719a483ab073d3d67...03975850c8240821a6dbe2e00750fb0fcc568e2a)
> [dcos/dcos-net#117](https://github.com/dcos/dcos-net/pull/118)
> https://circleci.com/gh/dcos/dcos-net/911
> https://codecov.io/github/dcos/dcos-net/commit/03975850c8240821a6dbe2e00750fb0fcc568e2a
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and review process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using GitHub's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by Fabs** [comment]
@mesosphere-mergebot label [Ready For Review]

**Comment by Fabs** [comment]
Could you please review @GoelDeepak, @urbanserj

File: gen/dcos-config.yaml [hunk 1]
```diff
@@ -2206,7 +2206,10 @@ package:
       # Read metrics from DC/OS Net Prometheus endpoint.
       [[inputs.prometheus]]
         ## An array of urls to scrape metrics from.
-        urls = ["http://127.0.0.1:62080/v1/metrics/default"]
+        urls = [
+          "http://127.0.0.1:62080/v1/metrics/default"
+          "http://127.0.0.1:62080/v1/metrics/dns"
```
Comment (urbanserj) [inline gen/dcos-config.yaml]: I would like to add `dns_` prefix for all dns metrics in telegraf, i.e.

```
[[inputs.prometheus]]
  ## An array of urls to scrape metrics from.
  urls = ["http://127.0.0.1:62080/v1/metrics/dns"]
  ## Specify timeout duration for slower prometheus clients (default is 3s)
  response_timeout = "30s"
  name_prefix = "dns_"
  ## Apply DC/OS component name tag according to the documentation.
  [inputs.prometheus.tags]
    dcos-component-name = "DC/OS Net"
```

Comment (Fabs) [reply gen/dcos-config.yaml]: Makes a lot of sense.

Comment (Fabs) [reply gen/dcos-config.yaml]: Should we also rename the dcos-component-name to "DC/OS Net DNS"?

Comment (urbanserj) [reply gen/dcos-config.yaml]: No, we shouldn't. The component is still called "DC/OS Net". Maybe we should add one more tag in the future, but at the moment let's keep it simple. We always can change it. That's exactly why I suggested to have registry per subproject.

Comment (Fabs) [reply gen/dcos-config.yaml]: Ok, I fixed the label back to "DC/OS Net".

File: CHANGES.md [hunk 2]
```diff
@@ -64,8 +64,11 @@ This change also aligned the authentication architectures between DC/OS Enterpri
 * Admin Router returns relative redirects to avoid relying on the Host header (DCOS-47845)
 
 * Introduced the `dcos-ui-update-service`, this component exposes an API to update the servered `dcos-ui` version using the `dcos-ui` package published to Universe.
+
 * Add basic support for prometheus to dcos-net (DCOS_OSS-4738)
 
+* Add Metrics for dns forwarding (DCOS-48336)
```
Comment (ichernetsky) [inline CHANGES.md]: nit: Add metrics for DNS forwarding

Comment (Fabs) [reply CHANGES.md]: Thanks, I will save it for next time I will touch this file because it will happen within the next RI.

**Pull request merged**